### ISSUE TITLE
Improve the display of the list

### ISSUE THREAD

**Issue by Maxou44** [mmozeiko/RcloneBrowser#7]
The display of the list in reduced mode is not very practical, I propose something like this:

![exemple](https://cloud.githubusercontent.com/assets/8138585/22446232/dcccbf8a-e74b-11e6-95cc-db905ec0590f.PNG)

**Comment by mmozeiko** [comment]
If I understand correctly, you are suggesting for each transfer to put speed and progress bar in collapsed view? Transfer speed is an easy change, but where does progress bar should get value from? rclone doesn't offer total progress for whole transfer.

**Comment by Maxou44** [comment]
Rclone offers the percentage for each upload, you can use the average of all percentages in each part ?

**Comment by mmozeiko** [comment]
What is (Sum of size files)? Its not a constant. If new files are added to folder, the size changes. If files are deleted, size changes. rclone CLI will be fine with it, it will simply skip deleted files or add newly added files to transfer.

And when downloading calculating total size of files can take potentially a lot of time. On my encrypted ACD remote for one large backup folder this takes more than 4 minutes.

I don't think I want to add extra operations like this, if rclone doesn't offer this information itself.

**Comment by Xalaxis** [comment]
I agree with @mmozeiko that progress on a non-per file basis is impossible, but support the speed view change.
